doc/source/actions.rst

```diff
@@ -214,3 +214,57 @@ The ``merge`` action merges the pull request into its base branch. The
        * GitHub branch protection of the contributor repository may refuse Mergify to
          force push the rebased pull request.
        * GPG signed commits will lost their signatures.
+
+Some advanced merge configuration example
```

Comment:
`Advanced Merge Configuration Examples`

Though is this examples or equivalence? If they are strictly equivalent, we could probably provide a table, that'd be clearer.